File: ios/Classes/IJKPlayerPlugin.m
#import "IJKPlayerPlugin.h"
#import "IJKPlayerFactory.h"
@implementation IJKPlayerPlugin
+ (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar>*)registrar {
    NSLog(@"viewId:1231323");
    IJKPlayerFactory* playerFactory = [[IJKPlayerFactory alloc] initWithMessenger:registrar.messenger];
    [registrar registerViewFactory:playerFactory withId:@"plugin.ugle.cn/piliplayer"];
    //[registrar publish:playerFactory];
}



@end

File: android/src/main/java/cn/ugle/plugin/ijkplayer/IJKPlayerPlugin.java
package cn.ugle.plugin.ijkplayer;

import io.flutter.plugin.common.MethodCall;
import io.flutter.plugin.common.MethodChannel;
import io.flutter.plugin.common.PluginRegistry;

/** IjkplayerPlugin */
public class IJKPlayerPlugin {
    /** Plugin registration. */
    public static void registerWith(PluginRegistry.Registrar registrar) {
        registrar.platformViewRegistry().registerViewFactory("plugin.ugle.cn/piliplayer", new IJKPlayerFactory(registrar.messenger()));
    }
}